torch_xla/csrc/tensor_impl.cpp

```diff
@@ -161,6 +161,11 @@ at::IntArrayRef XLATensorImpl::strides_custom() const {
   return strides_default();
 }
 
```

Comment:
You mentioned this `breaks log softmax lowering`. Can you add a test that will fail without this change? The change in general make sense but without a test it is hard to make sure this is correct nor can we prevent this from regressing again.